openlibrary/templates/lib/nav_head.html

```diff
@@ -1,27 +1,28 @@
 $def with (page)
 
 <header id="header-bar" class="header-bar">
-  <div class="hamburger-component header-dropdown">
-
-    <details>
-      <summary>
-        <img src="/static/images/menu.png" alt="additional options menu"/>
-      </summary>
-      <div class="hamburger-dropdown-component navigation-dropdown-component">
-        <ul class="dropdown-menu hamburger-dropdown-menu">
-          $if not ctx.user:
-            <li><a href="/account/login">$_("Log in")</a></li>
-            <li><a href="/account/create">$_("Sign up")</a></li>
-
-          <li><a href="/books/add">$_("Add a Book")</a></li>
-          <li><a href="/sponsorship">$_("Sponsor a Book")</a></li>
-          <li><a href="/recentchanges">$_("Recent Community Edits")</a></li>
-          <li><a href="/developers">$_("Developer Center")</a></li>
-          <li><a href="/help">$_("Help & Support")</a></li>
-        </ul>
-      </div>
-    </details>
-  </div>
+  $if not ctx.user:
```

Comment:
```suggestion
  $if ctx.user:
```
Login links not visible in hamburger menu when logged out (but visible when logged in).

Reply:
Done